eth/fetcher/block_fetcher.go

```diff
@@ -723,6 +723,9 @@ func (f *BlockFetcher) loop() {
 						matched = true
 						if f.getBlock(hash) == nil {
 							block := types.NewBlockWithHeader(announce.header).WithBody(task.transactions[i], task.uncles[i])
+							if block.Header().EmptyWithdrawalsHash() {
```

Comment:
set empty withdraws at fetcher side

Reply:
Do we need similar changes in downloader here? 
https://github.com/bnb-chain/bsc/blob/38db9bf4e29b00e5675d4cd9297b9a979ecea7b6/eth/downloader/downloader.go#L1392

Reply:
> Do we need similar changes in downloader here?
> 
> https://github.com/bnb-chain/bsc/blob/38db9bf4e29b00e5675d4cd9297b9a979ecea7b6/eth/downloader/downloader.go#L1392

esult.Withdrawals is already empty now

Reply:
Ok. But under what condition the above if statement of fetcher will be true?